shiny/ui/fill/_fill.py

```diff
@@ -215,3 +215,15 @@ def is_fill_item(tag: object) -> bool:
     # renders_to_tag_class(x, FILL_ITEM_CLASS, ".html-widget")
 
     return isinstance(tag, Tag) and tag.has_class(FILL_ITEM_CLASS)
+
+
+def add_fill_classes(
+    tag: Tag, fill_item: bool = True, fillable_container: bool = True
+) -> Tag:
+    if fill_item:
+        tag.add_class(FILL_ITEM_CLASS)
+    if fillable_container:
+        tag.add_class(FILL_CONTAINER_CLASS)
+    if fill_item or fillable_container:
+        tag.append(fill_dependency())
+    return tag
```

Comment:
With this being so close to the release, I'd rather use the existing `as_fill_item()`/`as_fillable_container()`. 

There is also the subtle `copy()` issue here, which I know isn't a problem in the way you're currently using it, but it'd be very easy for someone use this in future and unknowingly mutate user input

Reply:
I mean, the point is to avoid the unnecessary copy for elements we're creating internally. But OTOH I can see that it's worth avoiding the potential for mutating user input or having to document two separate approaches, plus it's only a shallow copy of the outer tag anyway. I'll go back to the existing functions.

Reply:
FTR, we actually make two copies because we need to call `as_fill_item(as_fill_carrier(tag))` and both make a copy.

Personally, I'd definitely prefer what @schloerke [recommended above](https://github.com/posit-dev/py-shiny/pull/1436#discussion_r1678415437): having `add_fill_classes()` for internal usage and also calling it directly in `as_fill_item()` and `as_fill_carrier()`. I think a reasonable docstring on `add_fill_classes()` would help with the mutating inputs issue.

Reply:
Anyway, 5bbe19ec82d9e211419bf2d701eedee0e61b083c reverts back to using `as_fill_item(as_fill_carrier(main))`